src/scripts/sort.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+function tableSort(type, tableItems) {
+  let sortedTable;
+
+  switch (type) {
+    case 'Name':
+      sortedTable = [...tableItems]
+        .sort((a, b) => (
+          a.children[0].innerText.localeCompare(b.children[0].innerText)
+        ));
+      break;
+
+    case 'Position':
+      sortedTable = [...tableItems]
+        .sort((a, b) => (
+          a.children[1].innerText.localeCompare(b.children[1].innerText)
+        ));
+      break;
+
+    case 'Office':
+      sortedTable = [...tableItems]
+        .sort((a, b) => (
+          a.children[2].innerText.localeCompare(b.children[2].innerText)
+        ));
+      break;
```

Comment:
Cool, but as for me, better to select compare function in this switch/case, and use sort only once